src/openloco/graphics/gfx.cpp

```diff
@@ -180,6 +183,343 @@ namespace openloco::gfx
+                case 12:
+                    _currentFontFlags = _currentFontFlags & ~TEXT_DRAW_FLAG_OUTLINE;
+                    break;
+                case 13:
+                {
+                    int hue = windowColours[0];
+                    setTextColours(colour::get_shade(hue, 7), 0x0A, 0x0A);
+                    break;
+                }
+                case 14:
+                {
+                    int hue = windowColours[1];
+                    setTextColours(colour::get_shade(hue, 9), 0x0A, 0x0A);
+                    break;
+                }
+                case 15:
+                {
+                    int hue = windowColours[2];
+                    setTextColours(colour::get_shade(hue, 9), 0x0A, 0x0A);
+                    break;
+                }
+
+                case 23:
+                {
+                    uint32_t image = ((uint32_t*)str)[0];
+                    uint32_t imageId = image & 0x7FFFF;
+                    str += 4;
+
+                    if ((_currentFontFlags & TEXT_DRAW_FLAG_OUTLINE) != 0)
+                    {
+                        gfx::draw_image_solid(context, pos.x, pos.y, imageId, _textColours[3]);
+                        gfx::draw_image_solid(context, pos.x + 1, pos.y + 1, imageId, _textColours[1]);
+                    }
+                    else
+                    {
+                        gfx::draw_image(context, pos.x, pos.y, image);
+                    }
+
+                    pos.x += _g1Elements[imageId].width;
+                    break;
+                }
+
+                case 144 + 0:
+                    // black
+
+                case 144 + 1:
+                    // grey
+
+                case 144 + 2:
+                    // white
+
+                case 144 + 3:
+                    // red
+
+                case 144 + 4:
+                    // green
+
+                case 144 + 5:
+                    // yellow
+
+                case 144 + 6:
+                    // TOPAZ
+
+                case 144 + 7:
+                    // CELADON
+
+                case 144 + 8:
+                    // BABYBLUE
+
+                case 144 + 9:
+                    // PALELAVENDER
+
+                case 144 + 10:
+                    // PALEGOLD
+
+                case 144 + 11:
+                    // LIGHTPINK
+
+                case 144 + 12:
+                    // PEARLAQUA
+
+                case 144 + 13:
+                    // PALESILVER
+                    {
+                        int c = chr - 144;
+                        auto el = &_g1Elements[2169];
+                        setTextColours(el->offset[c * 4 + 0], el->offset[c * 4 + 1], el->offset[c * 4 + 2]);
+                        break;
+                    }
+
+                default:
+                    if (chr >= 32)
+                    {
+
+                        gfx::draw_sprite_palete_set(context, pos.x, pos.y, 1116 + chr - 32 + _currentFontSpriteBase, _textColours);
```

Comment:
We might want to start mapping unicode points to sprites over here, I think? Not needed for this PR, but certainly when start using our own language files…

Reply:
Once we get to unicode support, I'd rather just remove sprite font support.

Reply:
I wouldn't be against that, as long as we ship with a (TrueType) font that mimics the sprite font as well, if only as an option.